packages/contractkit/src/wallets/aws-hsm-wallet.test.ts

```diff
@@ -0,0 +1,259 @@
+import { normalizeAddressWith0x } from '@celo/utils/lib/address'
+import { verifySignature } from '@celo/utils/lib/signatureUtils'
+import crypto from 'crypto'
+import { ec as EC } from 'elliptic'
+import Web3 from 'web3'
+import { EncodedTransaction, Tx } from 'web3-core'
+import { recoverTransaction, verifyEIP712TypedDataSigner } from '../utils/signing-utils'
+import AwsHsmWallet from './aws-hsm-wallet'
+import {
+  ACCOUNT_ADDRESS1,
+  ACCOUNT_ADDRESS2,
+  ACCOUNT_ADDRESS_NEVER,
+  CHAIN_ID,
+  TYPED_DATA,
+} from './test-utils'
+
```

Comment:
I believe you'll have to add this line to read from env file:
`require('dotenv').config()`